burger.css

```diff
@@ -0,0 +1,50 @@
+   .header__nav {
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    background: #E5E5E5;
```

Comment:
Codacy found an issue: [Expected "#E5E5E5" to be "#e5e5e5" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/MyBali/pullRequest?prid=5781003)